src/controller/ProjectManager.py

```diff
@@ -365,4 +382,13 @@ def import_raw_data(self, path: str):
         :param path: path where the raw_data is located
         """
         raw_data = DataFrame(FileManager.import_(path))
+        raw_data = raw_data.set_index(raw_data.columns[0])
```

Comment:
```suggestion
```
This line causes the first attribute to be interpreted as the line index.
This is false and causes problems when exporting.